v3.3/glfw: CreateWindow can run into GLFW_NO_WINDOW_CONTEXT but doesn't accept it

I have a program that opens and closes a lot of windows. This error popped up and asked to be reported.

```GLFW: Please report this bug in the Go package immediately.
panic: ErrorCode(65546): The specified window has no context

goroutine 1 [running, locked to thread]:
github.com/go-gl/glfw/v3.3/glfw.acceptError({0xc000323c28, 0x2, 0x40c425?})
        /home/joshpattman/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20221017161538-93cebf72946b/error.go:178 +0x1e5
github.com/go-gl/glfw/v3.3/glfw.CreateWindow(0xb7e800?, 0xc000588a80?, {0xc000756c00?, 0x1?}, 0x0, 0xc0000a46c0)
        /home/joshpattman/go/pkg/mod/github.com/go-gl/glfw/v3.3/glfw@v0.0.0-20221017161538-93cebf72946b/window.go:350 +0x169
github.com/gopxl/pixel/pixelgl.NewWindow.func1()
        /home/joshpattman/go/pkg/mod/github.com/gopxl/pixel@v1.0.0/pixelgl/window.go:164 +0x3d9
github.com/gopxl/pixel/pixelgl.NewWindow.CallErr.func3()
        /home/joshpattman/go/pkg/mod/github.com/faiface/mainthread@v0.0.0-20171120011319-8b78f0a41ae3/mainthread.go:74 +0x22
github.com/faiface/mainthread.Run(0xc0006081c0)
        /home/joshpattman/go/pkg/mod/github.com/faiface/mainthread@v0.0.0-20171120011319-8b78f0a41ae3/mainthread.go:44 +0xba
github.com/gopxl/pixel/pixelgl.Run(0xc000236090?)
        /home/joshpattman/go/pkg/mod/github.com/gopxl/pixel@v1.0.0/pixelgl/run.go:32 +0x45
github.com/JoshPattman/gym.BeginRenderLoop(...)
        /home/joshpattman/go/pkg/mod/github.com/!josh!pattman/gym@v0.0.2/rendering.go:15
main.main()
        /home/joshpattman/Documents/Coding/Go/ai/rl-cartpole/main.go:24 +0x358
exit status 2
```

--- a/v3.3/glfw/error.go
+++ b/v3.3/glfw/error.go
@@ -23,6 +23,7 @@ const (
 	invalidValue     ErrorCode = C.GLFW_INVALID_VALUE      // One of the parameters for the function was given an invalid value.
 	outOfMemory      ErrorCode = C.GLFW_OUT_OF_MEMORY      // A memory allocation failed.
 	platformError    ErrorCode = C.GLFW_PLATFORM_ERROR     // A platform-specific error occurred that does not match any of the more specific categories.
+	noWindowContext  ErrorCode = C.GLFW_NO_WINDOW_CONTEXT  // A window that does not have an OpenGL or OpenGL ES context was passed to a function that requires it to have one.
 )
 
 const (
@@ -84,6 +85,8 @@ func (e ErrorCode) String() string {
 		return "OutOfMemory"
 	case platformError:
 		return "PlatformError"
+	case noWindowContext:
+		return "NoWindowContext"
 	case APIUnavailable:
 		return "APIUnavailable"
 	case VersionUnavailable:
@@ -170,7 +173,7 @@ func acceptError(codes ...ErrorCode) error {
 	case platformError:
 		log.Println(err)
 		return nil
-	case notInitialized, noCurrentContext, invalidEnum, invalidValue, outOfMemory:
+	case notInitialized, noCurrentContext, invalidEnum, invalidValue, outOfMemory, noWindowContext:
 		panic(err)
 	default:
 		fmt.Fprintln(os.Stderr, "go-gl/glfw: internal error: an invalid error was not accepted by the caller:", err)

